OpenRA.Mods.RA/OpenRA.Mods.RA.csproj

```diff
@@ -213,9 +187,6 @@
   -->
   <PropertyGroup>
     <PostBuildEvent>mkdir "$(SolutionDir)mods/ra/"
-copy "$(TargetPath)" "$(SolutionDir)mods/ra/"
-cd "$(SolutionDir)thirdparty/"
-copy "FuzzyLogicLibrary.dll" "$(SolutionDir)"
-cd "$(SolutionDir)"</PostBuildEvent>
+copy "$(TargetPath)" "$(SolutionDir)mods/ra/"</PostBuildEvent>
   </PropertyGroup>
```

Comment:
This causes the build with MonoDevelop to fail, see https://github.com/OpenRA/OpenRA/pull/7422#issuecomment-72504436